### PR TITLE
readme: fix typos

### PR DESCRIPTION
**Pull request by vednoc**
The title says it all. 😄

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -120,7 +120,7 @@
 If you would like to contribute to this repository, please...
 
 1. [![fork](https://user-images.githubusercontent.com/136959/42383736-c4cb0db8-80fd-11e8-91ca-12bae108bccc.png) Fork](https://github.com/StylishThemes/GitHub-Dark/fork)
-1. Make sure you have these installed:
+2. Make sure you have these installed:
```
Comment (silverwind) [inline README.md:123]: Not strictly a typo because this 1., 1., 1. numbering is a feature of markdown which saves the user from having to count and renumber when adding to the list.

File: README.md [hunk 2]
```diff
@@ -167,8 +165,12 @@ Internal use only targets include:
...
-- If you're using a custom domain for GitHub Enterprise, be sure to include it through a `@-moz-document` rule (Firefox) or add it to the `Applies to` section in (Chrome). See
+- If you're using a custom domain for GitHub Enterprise, be sure to include it through a `@-moz-document` rule (Firefox) or add it to the `Applies to` section in (Chrome).
 
 Thanks to all our [contributors](./AUTHORS) so far! And thanks for the shoutout on the [JS Party](https://changelog.com/jsparty/20#transcript-71) podcast!
 
 [⬆️ UP](#toc)
+
+:before:after:
+:------:-----:
+:![image](https://user-images.githubusercontent.com/18245694/91337952-73cb2e80-e7d4-11ea-8e8c-5bb624c0af43.png):![image](https://user-images.githubusercontent.com/18245694/91338030-952c1a80-e7d4-11ea-948d-58b52c761b90.png):
```
Comment (silverwind) [inline README.md:176]: This seems unrelated.

File: README.md (hunk 1 above)
Comment (vednoc) [reply README.md:123]: Agreed, but in plain text it looks like a typo. _Installation steps_ also use proper numbers.

File: README.md (hunk 2 above)
Comment (vednoc) [reply README.md:176]: Yep! I tried to format a table in my text editor and forgot to remove it. 😅

File: README.md (hunk 1 above)
Comment (silverwind) [reply README.md:123]: Yeah fine here I guess
